setup1-contributor/src/commands/contribute.rs

```diff
@@ -954,9 +954,9 @@ impl StatusUpdater {
                 self.update_position_in_round(&progress_bar).await?;
             }
             ContributorStatus::Other => {
-                progress_bar.set_message(
-                    &"Not in the queue for Aleo Setup ceremony. Please double check the address \
-                    you are connecting, then disconnect and try again",
+                progress_bar.finish_with_message(
+                    "Not in the queue for Aleo Setup ceremony. Please double check the address \
+                    you are connecting to, then disconnect and try again",
```

Comment:
Hmm, actually, if this only comes up at the end of the round (from the code I'm assuming we'll always be in the queue before the bar comes up), should this message be more centered around the contributor's work being finished?

Reply:
we can probably add one more status? we can check if the contributor is one of the finished contributors

Reply:
Makes sense!